xbmc/interfaces/legacy/ModuleXbmc.h

```diff
@@ -157,6 +157,9 @@ namespace XBMCAddon
     /// Execute a built in Kodi function.
     ///
     /// @param function                string - builtin function to execute.
+    /// @param wait                    [opt] bool - if true wait until execute is
```

Comment:
[opt] bool - If Kodi should wait for the builtin function execution to finish (default False)

Reply:
Thanks a lot have it update.